libcnb-data/src/launch.rs

```diff
@@ -168,4 +241,52 @@ mod tests {
             Err(ProcessTypeError::InvalidValue(String::new()))
         );
     }
+
+    #[test]
+    fn process_with_default_values_deserialization() {
+        let toml_str = r#"
+type = "web"
+command = "foo"
+"#;
+
+        assert_eq!(
+            toml::from_str::<Process>(toml_str),
+            Ok(Process {
+                r#type: process_type!("web"),
+                command: String::from("foo"),
+                args: vec![],
+                direct: false,
+                default: false
+            })
+        );
+    }
+
+    #[test]
+    fn process_with_default_values_serialization() {
+        let process = ProcessBuilder::new(process_type!("web"), "foo").build();
+
+        let string = toml::to_string(&process).unwrap();
+        assert_eq!(
+            string,
+            r#"type = "web"
+command = "foo"
+"#
+        );
+    }
+
+    #[test]
+    fn process_with_some_default_values_serialization() {
+        let process = ProcessBuilder::new(process_type!("web"), "foo")
+            .default(true)
+            .build();
+
+        let string = toml::to_string(&process).unwrap();
+        assert_eq!(
+            string,
+            r#"type = "web"
+command = "foo"
+default = true
+"#
+        );
+    }
```

Comment:
These are tests that should've been added with #243. Since this PR changes the implementation form #243 slightly, I felt obligated to add them with this PR.